tests/ert/ui_tests/gui/test_restart_ensemble_experiment.py

```diff
@@ -147,3 +148,139 @@ def handle_dialog():
     assert (
         list_model.rowCount() == experiment_panel.config.runpath_config.num_realizations
     )
+
+
+def test_restart_failed_realizations_evaluate_ensemble(
```

Comment:
The test looks fine. Just wondering if it is really necessary to run 100 realizations and not just 10?